### PR TITLE
[doc][yba] Update backup flags

### PR DESCRIPTION
**Pull request by ddhodge**
Update YBA Installer backup flags

@netlify /preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer/

### CODE REVIEW

**Comment by dataindataout** [comment]
Thank you -- this addition is great.

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 1]
```diff
@@ -48,51 +48,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
...
-    | --verbose | Display extra information in the output. (default: false) |
-    | -f, --force (global flag) | Run in non-interactive mode. All user confirmations are skipped. |
-    | --log_level string (global flag) | Log level for this command. Levels: panic, fatal, error, warn, info, debug, trace. (default: "info") |
+    ```sh
+    $ sudo yba-ctl createBackup ~/test_backup
+    $ ls test_backup/
+    ```
+
+    ```output
+    backup_23-04-25-16-54.tgz  version_metadata_backup.json
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:61]: I don't think we want this? Where did version_metadata_backup.json come from that shouldn't be there

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 2]
```diff
@@ -48,51 +48,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
...
-| --skip_dbdrop | Skip dropping the yugaware database before a migration restore. (default: false) |
-| --skip_restart | Don't restart processes during command execution. (default: false) |
-| --use_system_pg | Use system path's `pg_restore` as opposed to installed binary. (default: false) |
-| --verbose | Display extra information in the output. (default: false) |
-| -f, --force (global flag) | Run in non-interactive mode. All user confirmations are skipped. |
-| --log_level string (global flag) | Log level for this command. Levels: panic, fatal, error, warn, info, debug, trace. (default: "info") |
\ No newline at end of file
+| Flag | Description | Default |
+| :--- | :---------- | :------ |
+| --destination | Path to un-tar the backup. | "/opt/yugabyte" |
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:112]: Same with this yba-ctl. The default also isn't necessarily /opt/yugabyte but whatever they set for installRoot. We should probably just let yba-ctl figure this out it's more likely to go wrong if customer specifies this

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 3]
```diff
@@ -48,51 +48,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
...
+| -h, --help | Help for `restoreBackup`. | |
+| --migration | Restore from a Replicated installation. | false |
+| --skip_dbdrop | Skip dropping the YugabyteDB Anywhere database before a migration restore. | false |
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:115]: maybe highlight this is only valid option for a migration, if specified without --migration it is irrelevant.

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:61]: version_metadata_backup.json comes from here https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/#backup-and-restore
If that is incorrect, then we need to change it there as well

File: docs/content/stable/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 4]
```diff
@@ -48,51 +48,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
...
 
     ```sh
     sudo yba-ctl restoreBackup <input_path> [flags]
     ```
 
-    The `restoreBackup` command executes the `yb_platform_backup.sh` script to restore from a previously taken backup of your YugabyteDB Anywhere instance. Specify the `input_path` to the backup `.tar.gz` file as the only argument.
+    Specify the `input_path` to the backup file as the only argument. For example:
+
+    ```sh
+    $ sudo yba-ctl restoreBackup ~/test_backup/backup_23-04-25-16-64.tgz
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:100]:
```suggestion
sudo yba-ctl restoreBackup ~/test_backup/backup_23-04-25-16-64.tgz
```

File: docs/content/v2.18/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 5]
```diff
@@ -47,51 +47,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
...
     ```
 
-    The `createBackup` command executes the `yb_platform_backup.sh` script to create a backup of your YugabyteDB Anywhere instance. Specify the `output_path` where you want the backup `.tar.gz` file to be stored.
+    The `createBackup` command executes the `yb_platform_backup.sh` script to create a backup of your YugabyteDB Anywhere instance. Specify the `output_path` where you want the backup `.tgz` file to be stored.
 
-    The following table describes optional flags you can include with the `createBackup` command.
+    The `createBackup` command creates a timestamped `tgz` file for the backup. For example:
+
+    ```sh
+    $ sudo yba-ctl createBackup ~/test_backup
```
Comment (aishwarya24) [inline docs/content/v2.18/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:55]:
```suggestion
sudo yba-ctl createBackup ~/test_backup
```

File: docs/content/v2.18/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 6]
```diff
@@ -47,51 +47,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
...
+    $ ls test_backup/
```
Comment (aishwarya24) [inline docs/content/v2.18/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:56]:
```suggestion
 ls test_backup/
```

File: docs/content/v2.18/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 7]
```diff
@@ -47,51 +47,73 @@ To back up your YugabyteDB Anywhere installation, perform the following steps:
...
 
     ```sh
     sudo yba-ctl restoreBackup <input_path> [flags]
     ```
 
-    The `restoreBackup` command executes the `yb_platform_backup.sh` script to restore from a previously taken backup of your YugabyteDB Anywhere instance. Specify the `input_path` to the backup `.tar.gz` file as the only argument.
+    Specify the `input_path` to the backup file as the only argument. For example:
+
+    ```sh
+    $ sudo yba-ctl restoreBackup ~/test_backup/backup_23-04-25-16-64.tgz
```
Comment (aishwarya24) [inline docs/content/v2.18/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:99]:
```suggestion
sudo yba-ctl restoreBackup ~/test_backup/backup_23-04-25-16-64.tgz
```

Comment (aishwarya24) [reply docs/content/v2.18/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:99]: Remove the $ anywhere else too.

**Review by aishwarya24 — COMMENTED** [review]
minor comments

**Pull request merged**